src/unstract/sdk/ocr.py

```diff
@@ -8,47 +8,58 @@
 
 from unstract.sdk.adapters import ToolAdapter
 from unstract.sdk.constants import LogLevel
+from unstract.sdk.exceptions import OCRError
 from unstract.sdk.tool.base import BaseTool
 
 
 class OCR(metaclass=ABCMeta):
     def __init__(
         self,
         tool: BaseTool,
-        adapter_instance_id: str,
+        adapter_instance_id: str = None,
```

Comment:
@gaya3-zipstack please correct this type to `Optional[str]` (at all other places too)

Reply:
Thank you!! Done.